### DISCUSSION TITLE
RCON syntax has changed

### DISCUSSION THREAD

**Discussion by abesnier** [itzg/docker-minecraft-server#1519] [Q&A] [answered]
Hi All, 

I tried to add some gamerules in my docker-compose file. I followed the example from the readme (with Paper and minecraft 1.18.2):
```
      RCON_CMDS_STARTUP:  |-
        /gamerule fallDamage false
        /gamerule drowningDamage false
        /gamerule fireDamage false
        /gamerule freezeDamage false
```
(don't judge, it's a server for my young kids!)

For the love of me, I could not understand why this failed with the following errors:
```
[Rcon loop] running - /gamerule freezeDamage false
[07:37:05 INFO]: Thread RCON Client /127.0.0.1 started
[07:37:05 INFO]: Thread RCON Client /127.0.0.1 shutting down
[Rcon loop] Unknown command. Type "/help" for help.
```

Well, I am really not familiar with RCON, but the issue is that commands do not need to start with /.
Removing the / was the way to go. Maybe updating the documentation would help?

For info, I tried running `docker exec -it minecraft rcon-cli`, and here's a sample of my interaction:

```
me@jmycomputer:/srv/minecraft/minecraft# docker exec -it minecraft  rcon-cli
> /help
Unknown command. Type "/help" for help.

> help
--------- Help: Index (1/10) --------------------
Use /help [n] to get page n of help.
Aliases: Lists command aliases
Bukkit: All commands for Bukkit
Minecraft: All commands for Minecraft
/advancement: A Mojang provided command.
/attribute: A Mojang provided command.
/ban: A Mojang provided command.
/ban-ip: A Mojang provided command.
/banlist: A Mojang provided command.

> help Minecraft
--------- Help: Minecraft (1/9) -----------------
Below is a list of all Minecraft commands:
/advancement: A Mojang provided command.
/attribute: A Mojang provided command.
/ban: A Mojang provided command.
/ban-ip: A Mojang provided command.
/banlist: A Mojang provided command.
/bossbar: A Mojang provided command.
/clear: A Mojang provided command.
/clone: A Mojang provided command.

> ban     
Unknown or incomplete command, see below for errorban<--[HERE]
> /ban
Unknown command. Type "/help" for help.
```
So basically, / is not recognized anymore?

if someone more versed than me in Minecraft and RCON can explain what happened, please do!

Cheers

**Comment by itzg** [comment, marked as answer]
I would have suggested removing the leading slash. I wrote https://github.com/itzg/docker-minecraft-server/issues/1520 to investigate/fix that.
